systemd/systemd.go

```diff
@@ -392,6 +392,7 @@ WantedBy={{.ServiceSystemdTarget}}
 		filepath.Join(desc.SnapPath, desc.Start),
 		filepath.Join(desc.SnapPath, desc.Stop),
 		filepath.Join(desc.SnapPath, desc.PostStop),
+		// XXX: do we still need to offer a SNAP_APP=AppTriple??
```

Comment:
No, that's bogus. The format doesn't match anything in use now or even before. I'd just drop it altogether, and if we need to reintroduce SNAP_APP in the future, it should be the app name alone.